crates/wash-lib/src/context/fs.rs

```diff
@@ -190,24 +233,24 @@ mod test {
             .save_context(&orig_ctx)
             .expect("Should be able to save a context to disk");
 
-        let mut readdir = contexts_path.read_dir().unwrap();
-        let ctx_path = readdir
-            .next()
-            .expect("Should be at least 1 entry in directory")
+        let filenames: std::collections::HashSet<String> = contexts_path
+            .read_dir()
             .unwrap()
-            .path();
+            .filter_map(|entry| entry.unwrap().file_name().to_os_string().into_string().ok())
+            .collect();
+        let expected_filenames: std::collections::HashSet<String> = vec![
+            "default".to_string(),
+            "host_config.json".to_string(),
+            "happy_path.json".to_string(),
+        ]
+        .into_iter()
+        .collect();
```

Comment:
```suggestion
        let expected_filenames = std::collections::HashSet::from([
            "default".to_string(),
            "host_config.json".to_string(),
            "happy_path.json".to_string(),
        ]);
```

Reply:
I tried this and was confused why it didn't work. It's because I had the `vec!` 🙂